website/docs/r/traffic_mirror_filter.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Traffic Mirroring"
+layout: "aws"
+page_title: "AWS: aws_traffic_mirror_filter"
+sidebar_current: "docs-aws-resource-traffic-mirror-filter"
```

Comment:
The `sidebar_current` frontmatter has been deprecated and is no longer necessary (re-running CI will now return an error if this is still present)